int32 first_digit
int32 second_digit
